Tests/TracingTests/TestTracer.swift

```diff
@@ -43,6 +43,12 @@ final class TestTracer: LegacyTracer {
         return span
     }
 
+    /// Export all ended spans to the configured backend that have not yet been exported.
+    ///
+    /// This function should only be called in cases where it is absolutely necessary,
+    /// such as when using some FaaS providers that may suspend the process after an invocation, but before the backend exports the completed spans.
+    ///
+    /// This function should not block indefinitely, implementations should offer a configurable timeout for flush operations.
```

Comment:
Same here, IMHO not worth documenting on a test impl, this comes from a protocol.